R/dash.R

```diff
@@ -920,7 +912,7 @@ Dash <- R6::R6Class(
 # @param layout
 # @param component a component (should be a dependency)
 validate_dependency <- function(layout_, dependency) {
-  if (!is.layout(layout_)) stop("`layout` must be a dash layout object", call. = FALSE)
+  if (!is.component(layout_)) stop("`layout` must be a Dash component or collection of components", call. = FALSE)
```

Comment:
Is there a case we can actually hit this `stop`? Doesn't the one in `layout_render` cover us?

Reply:
I have trouble seeing where this gets tripped as well; this is leftover code from the early revisions of the package. I suspect it made more sense when we wrapped the layout in an `htmlDiv` and wanted to ensure the layout was a list. 

Here I'm actively trying to avoid that, so it probably does make sense to try :hocho: this.